Extensions/Leaderboards/leaderboardstools.ts

```diff
@@ -105,6 +105,8 @@ namespace gdjs {
       _loaderContainer.style.width = '100%';
       _loaderContainer.style.justifyContent = 'center';
       _loaderContainer.style.alignItems = 'center';
+      _loaderContainer.style.position = 'absolute';
```

Comment:
Is the `absolute` needed?

Reply:
Z-index must be used with "positioned box". I am not sure if there are other positions that are more "neutral" than absolute and that is not "static" ?